src/charts/Chart.Radar.js

```diff
@@ -1,15 +1,9 @@
 "use strict";
 
 module.exports = function(Chart) {
-
-	var helpers = Chart.helpers;
-
-	var defaultConfig = {
```

Comment:
`Chart.Scatter` also define [defaultConfig](https://github.com/chartjs/Chart.js/blob/master/src/charts/Chart.Scatter.js#L5), do we really want to inline `defaultConfig` here?